man/oci-runtime-tool-generate.1.md

```diff
@@ -29,15 +29,15 @@ read the configuration from `config.json`.
 
   --args "/usr/bin/httpd" --args "-D" --args "FOREGROUND"
 
-**--bind**=*[[HOST-DIR:CONTAINER-DIR][:OPTIONS]]*
+**--bind**=*[[HOST-DIR:CONTAINER-DIR][:OPTIONS:...]]*
```

Comment:
I think most commands have the `...` suffix if you can specify multiple of a type. But I'll change it.